### ISSUE TITLE
Problem removing OneClick user

### ISSUE THREAD

**Issue by fceruti** [cornershop/python-tbk#9]
When running `remove_user` from `OneClickPaymentService` class, I get the following error: 

```bash
  File "XXX/site-packages/tbk/services.py", line 65, in remove_user
    return self.soap_requestor.request('oneClickRemoveUser', one_click_remove_user_input)
  File "XXX/site-packages/tbk/soap/requestor.py", line 74, in request
    **kwargs)
  File "XXX/site-packages/tbk/soap/zeep_client.py", line 35, in request
    method = self.get_method(method_name)
  File "XXX/site-packages/tbk/soap/zeep_client.py", line 51, in get_method
    raise MethodDoesNotExist(method_name)
tbk.soap.exceptions.MethodDoesNotExist: oneClickRemoveUser
```

The [documentation](https://www.transbankdevelopers.cl/referencia/webpay#eliminar-una-inscripcion-webpay-oneclick) shows that to perform this action you should use `removeUser` soap method and not `oneClickRemoveUser`.

**Comment by pedroburon** [comment]
Already merged!

**Comment by fceruti** [comment]
What do you mean with that comment? Was I supposed to close this issue?

**Comment by pedroburon** [comment]
> What do you mean with that comment? Was I supposed to close this issue?

this fix was merged on release v0.2.2
https://pypi.org/project/python-tbk/0.2.2/

Thanks!
